src/dynamics/mpas/Makefile.in.CESM

```diff
@@ -0,0 +1,110 @@
+ifeq ($(strip $(CASEROOT)),)
+    $(warning `CASEROOT` should not be empty. Defaulting to `.`)
+
+    CASEROOT = .
+endif
+
+ifeq ($(strip $(LIBROOT)),)
+    $(warning `LIBROOT` should not be empty. Defaulting to `..`)
+
+    LIBROOT = ..
+endif
+
+#
+# Define and export variables used by MPAS build infrastructure.
+#
+
+export CP = cp -afv
+export RM = rm -frv
+
+# Constants.
+export AUTOCLEAN       = false
+export BUILD_TARGET    = N/A
+export CORE            = atmosphere
+export EXE_NAME        = atmosphere_model
+export GEN_F90         = false
+export GIT_VERSION     = N/A
+export NAMELIST_SUFFIX = atmosphere
+
+# Read variables (e.g., build options) from `Macros.make`.
+include $(CASEROOT)/Macros.make
+
+# Translate variables (e.g., build options) from `Macros.make` to their equivalent ones used by MPAS build infrastructure.
+export AR             := ar
+export ARFLAGS        := -M
+export CC             := $(strip $(MPICC))
+export CFLAGS         := $(strip $(CFLAGS))
+export CPP            := cpp -P -traditional
+export CPPFLAGS       := -D_MPI \
+                         -DMPAS_BUILD_TARGET="$(BUILD_TARGET)" \
+                         -DMPAS_CAM_DYCORE \
+                         -DMPAS_EXE_NAME="$(EXE_NAME)" \
+                         -DMPAS_EXTERNAL_ESMF_LIB \
+                         -DMPAS_GIT_VERSION="$(GIT_VERSION)" \
+                         -DMPAS_NAMELIST_SUFFIX="$(NAMELIST_SUFFIX)" \
+                         -DMPAS_NATIVE_TIMERS \
+                         -DMPAS_NO_ESMF_INIT \
+                         -DMPAS_PIO_SUPPORT \
+                         -DUSE_PIO2
+export CPPINCLUDES    := $(strip $(CPPINCLUDES))
+export CXX            := $(strip $(MPICXX))
+export CXXFLAGS       := $(strip $(CXXFLAGS))
+export FC             := $(strip $(MPIFC))
+export FCINCLUDES     := $(strip $(FCINCLUDES))
+export FFLAGS         := $(strip $(FC_AUTO_R8) $(FFLAGS) $(FREEFLAGS))
```

Comment:
If you use the CIME Makefile then I believe `FFLAGS` and `FREEFLAGS` are already combined within the `FFLAGS` variable.

Reply:
Should be addressed in https://github.com/ESCOMP/CAM-SIMA/commit/9842660369bcbead473656860787ec430a28194a. Rely on CIME Makefile to handle this.